### PR TITLE
[devtools-translate] Russian - What's New in DevTools (Chrome 107)

### LINKED ISSUE #3756: [devtools-translate] Russian - What's New in DevTools (Chrome 107)
Post: https://developer.chrome.com/blog/new-in-devtools-107/
Translated by: Oct-11

Translator: @solarrust
Reviewers: @webmaxru, @kateryna-prokopenko

### PR DESCRIPTION
**Pull request by solarrust**
Fixes #3756

### CODE REVIEW

**Comment by jecfish** [comment]
Thanks @solarrust! Chrome 107 will roll out on Oct 25. Kateryna is OOO, appreciate if @webmaxru can review in these few days and we get this PR merge by this week!

File: site/ru/_partials/devtools/whats-new.md [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
-* [Highlight C/C++ objects in the Memory Inspector](/ru/blog/new-in-devtools-107/#memory)
-* [Support full initiator information for HAR import](/ru/blog/new-in-devtools-107/#har)
-* [Start DOM search after pressing `Enter`](/ru/blog/new-in-devtools-107/#search-type)
-* [Display `start` and `end` icons for `align-content` CSS flexbox properties](/ru/blog/new-in-devtools-107/#flexbox)
-* [Miscellaneous highlights](/ru/blog/new-in-devtools-107/#misc)
- -->
+### Chrome 107 {: #chrome107 }
+
+* [Кастомизация сочетаний клавиш в DevTools](/ru/blog/new-in-devtools-107/#shortcuts)
+* [Переключение между светлой и тёмной темами по сочитанию клавиш](/ru/blog/new-in-devtools-107/#toggle-themes)
```
Comment (webmaxru) [inline site/ru/_partials/devtools/whats-new.md:11]:
```suggestion
* [Переключение между светлой и тёмной темами по сочетанию клавиш](/ru/blog/new-in-devtools-107/#toggle-themes)
```

File: site/ru/blog/new-in-devtools-107/index.md [hunk 2]
```diff
@@ -21,102 +20,122 @@ draft: true
...
+  + 4. Translate all the <img> alt text
   5. Update the whats-new.md file -->
 
 <!-- Content starts here -->
 
 <!-- ## Customize keyboard shortcuts in DevTools {: #shortcuts } -->
+## Кастомизация сочетаний клавиш в DevTools {: #shortcuts }
 
 <!-- You can now customize keyboard shortcuts for your favorite commands in DevTools. -->
+Теперь вы можете кастомизировать горячие клавиши для ваших любимых комманд в DevTools.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-107/index.md:35]:
```suggestion
Теперь вы можете кастомизировать горячие клавиши для ваших любимых команд в DevTools.
```

File: site/ru/blog/new-in-devtools-107/index.md [hunk 3]
```diff
@@ -21,102 +20,122 @@ draft: true
...
 
 <!-- Go to **Settings** > **Shortcuts**, hover over a command and click the **Edit** button (pen icon) to customize the keyboard shortcut. You can create chords (a.k.a multi-key press shortcuts) as well.  -->
+Перейдите в **Настройки** (Settings) > **Сочетания клавиш** (Shortcuts), наведите курсор на команду и нажмите кнопку **Редактировать** (иконка карандаша) чтобы задать своё сочетание клавиш. Также вы можете задавать команды, состоящие из нескольких клавиш.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-107/index.md:38]:
```suggestion
Перейдите в **Настройки** (Settings) > **Сочетания клавиш** (Shortcuts), наведите курсор на команду и нажмите кнопку **Редактировать** (иконка карандаша), чтобы задать своё сочетание клавиш. Также вы можете задавать команды, состоящие из нескольких клавиш.
```

File: site/ru/blog/new-in-devtools-107/index.md [hunk 4]
```diff
@@ -21,102 +20,122 @@ draft: true
...
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/973EfWpxwGOdEF1nN1vv.png", alt="Кастомизация сочетаний клавиш в DevTools", width="800", height="516" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/d061128ff63a97ab2c6c0d2b5e655e6fcbed829c #}
 
-Chromium issues: [1335274](https://crbug.com/1335274), [174309](https://crbug.com/174309)
+Задачи в трекере Chromium: [1335274](https://crbug.com/1335274), [174309](https://crbug.com/174309)
 
 
 <!-- ## Toggle light and dark themes with keyboard shortcut {: #toggle-themes } -->
+## Переключение между светлой и тёмной темами по сочитанию клавиш {: #toggle-themes }
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-107/index.md:48]:
```suggestion
## Переключение между светлой и тёмной темами по сочетанию клавиш {: #toggle-themes }
```

File: site/ru/blog/new-in-devtools-107/index.md [hunk 5]
```diff
@@ -21,102 +20,122 @@ draft: true
...
 
 <!-- Configure a keyboard shortcut to toggle [light and dark themes](/docs/devtools/rendering/emulate-css/#emulate-css-media-feature-prefers-color-scheme) conveniently. By default, the action doesn’t map to any keyboard shortcut. -->
+Настройте сочетания клавиш для удобного переключения между [светлой и тёмной темами](/docs/devtools/rendering/emulate-css/#emulate-css-media-feature-prefers-color-scheme). По умолчанию для этого действия не настроено сочетание клавиш.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/7oGdE2eRsgwokWXW9XvA.png", alt="Toggle light and dark themes with keyboard shortcut.", width="800", height="576" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/7oGdE2eRsgwokWXW9XvA.png", alt="Переключение между светлой и тёмной темами по сочитанию клавиш", width="800", height="576" %}
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-107/index.md:53]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/7oGdE2eRsgwokWXW9XvA.png", alt="Переключение между светлой и тёмной темами по сочетанию клавиш", width="800", height="576" %}
```

File: site/ru/blog/new-in-devtools-107/index.md [hunk 6]
```diff
@@ -21,102 +20,122 @@ draft: true
...
 
 
 <!-- ## Highlight C/C++ objects in the Memory Inspector {: #memory } -->
+## Подсветка объектов C/C++ в Инспекторе памяти (Memory Inspector) {: #memory }
 
 <!-- The [Memory Inspector](/docs/devtools/memory-inspector/) highlights all the bytes of a C/C++ memory object. -->
+[Инспектор памяти](/docs/devtools/memory-inspector/) подсвечивает все байты объекта памяти C/C++.
 
 <!-- Recognizing an object’s bytes among the surrounding WebAssembly memory was a pain point. You have to know the object’s size and count bytes from the object’s start. -->
+Распознавание байтов объекта в окружении памяти WebAssembly — болезненная вещь. Вам нужно знать размер объекта и считать байты от его начала.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-107/index.md:68]:
```suggestion
Распознавание байтов объекта в окружении памяти WebAssembly — нетривиальная задача. Вам нужно знать размер объекта и считать байты от его начала.
```

File: site/ru/blog/new-in-devtools-107/index.md [hunk 7]
```diff
@@ -21,102 +20,122 @@ draft: true
...
 
 <!-- With this feature,  it helps you tell them apart from the surrounding memory. See [Extending the Memory Inspector for C/C++ debugging](/blog/memory-inspector-extended-cpp/) to learn more about the changes. -->
+Эта функция поможет вам отличить их от окружающей памяти. Читайте [Extending the Memory Inspector for C/C++ debugging](/blog/memory-inspector-extended-cpp/) чтобы узнать о других измнениях.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-107/index.md:71]:
```suggestion
Эта функция поможет вам отличить их от окружающей памяти. Читайте [Extending the Memory Inspector for C/C++ debugging](/blog/memory-inspector-extended-cpp/), чтобы узнать о других изменениях.
```

File: site/ru/blog/new-in-devtools-107/index.md [hunk 8]
```diff
@@ -21,102 +20,122 @@ draft: true
...
 
-Chromium issue: [1336568](https://crbug.com/1336568)
+Задача в трекере Chromium: [1336568](https://crbug.com/1336568)
 
 
 <!-- ## Support full initiator information for HAR import {: #har } -->
+## Поддержка полной информации об инициаторе для импорта HAR {: #har }
 
 <!-- Full **Initiator** information is available now for [HAR import](/docs/devtools/network/reference/#save-as-har). Previously, the **Network** panel only shows partial initiator information during import. -->
+Полная информация о **Инициаторе** (Initiator) теперь доступна для [импорта HAR](/docs/devtools/network/reference/#save-as-har). Ранее во вкладке **Сеть** (Network) показывалась только частичная информация о инициаторе при импорте.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-107/index.md:85]:
```suggestion
Полная информация об **Инициаторе** (Initiator) теперь доступна для [импорта HAR](/docs/devtools/network/reference/#save-as-har). Ранее во вкладке **Сеть** (Network) показывалась только частичная информация об инициаторе при импорте.
```

File: site/ru/blog/new-in-devtools-107/index.md [hunk 9]
```diff
@@ -21,102 +20,122 @@ draft: true
...
+Теперь вы можете отключить настройку **Поиск при вводе запроса** (Search as you type) чтобы всегда начинать поиск по DOM только после нажатия на <kbd>Enter</kbd>.
 
 <!-- In the **Elements** panel, toggle the search bar with <kbd>Control</kbd> or <kbd>Command</kbd> + <kbd>F</kbd>. As you type a query in the search textbox, the DOM tree will jump to the first matching element and highlight it by default.  -->
+Вызовите строку поиска во вкладке **Элементы** (Elements) при помощи <kbd>Control</kbd> или <kbd>Command</kbd> + <kbd>F</kbd>.
 
 <!-- For users, especially testers who always work with lengthy search queries, this behavior is not ideal. The DOM tree might jump multiple times as you type in a lengthy search query (e.g. `//div[@id="example"]`). This behavior creates unnecessary motion. -->
+Для пользователей, особенно инженеров тестирования, которые всегда работают с длинными поисковыми запросами, такое поведение далеко от идеала. Дерево DOM может прыгать множество раз пока вы набираете длинный поисковый запрос (например, `//div[@id="example"]`). Такое поведение создает ненужные скачки.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-107/index.md:107]:
```suggestion
Для пользователей, особенно инженеров по тестированию, которые всегда работают с длинными поисковыми запросами, такое поведение далеко от идеала. Дерево DOM может прыгать много раз, пока вы набираете длинный поисковый запрос (например, `//div[@id="example"]`). Такое поведение создает ненужные скачки.
```

File: site/ru/blog/new-in-devtools-107/index.md [hunk 10]
```diff
@@ -21,102 +20,122 @@ draft: true
...
 
 
 <!-- ## Display `start` and `end` icons for `align-content` CSS flexbox properties {: #flexbox } -->
+## Отображение иконок `start` и `end` для свойства флексбокса `align-content` {: #flexbox }
 
 <!-- In the **Styles** pane, edit the `align-content` properties in a CSS class with `display: flex` or `display: inline-flex`. The `start` and `end` show in the auto-complete dropdown with icons. -->
+Во вкладке **Стили** (Styles) отредактируйте свойство `align-content` у элемента с `display: flex` или `display: inline-flex`. Значения `start` и `end` будут отображаться с иконками в выпадающем меню автодополнения.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fo10I2mt6bQ357itnYhl.png", alt="align-content flexbox properties.", width="800", height="424" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fo10I2mt6bQ357itnYhl.png", alt="свойство флексбокса align-content", width="800", height="424" %}
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-107/index.md:127]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fo10I2mt6bQ357itnYhl.png", alt="Свойство флексбокса align-content", width="800", height="424" %}
```

File: site/ru/blog/new-in-devtools-107/index.md [hunk 11]
```diff
@@ -21,102 +20,122 @@ draft: true
...
 
-Chromium issue: [1139945](https://crbug.com/1139945)
+Задача в трекере Chromium: [1139945](https://crbug.com/1139945)
 
 
 <!-- ## Miscellaneous highlights {: #misc } -->
+## Другие важные моменты {: #misc }
 
 <!-- - Display correct message counts in the **Console** sidebar. Previously, the counts didn't refresh when clearing console messages. ([1343311](https://crbug.com/1343311)) -->
+- Отображение правильного счётчика сообщений в боковой панели **Консоли** (Console). Ранее счётчик не сбрасывался когда очищались сообщения в консоли. ([1343311](https://crbug.com/1343311))
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-107/index.md:138]:
```suggestion
- Отображение правильного счётчика сообщений в боковой панели **Консоли** (Console). Ранее счётчик не сбрасывался, когда очищались сообщения в консоли. ([1343311](https://crbug.com/1343311))
```

**Review by jecfish — APPROVED** [review]

**Comment by jecfish** [comment]
Merge this one as well! Thanks both!

**Pull request merged**